pages/app.css

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+body {
+    font-family: 'Inter', sans-serif;
+}
```

Comment:
once you have followed the hints for font optimization, you can delete this file

Reply:
Done :)